fea-rs/src/compile/lookups/contextual.rs

```diff
@@ -117,26 +117,27 @@ impl<T> ContextualLookupBuilder<T> {
             .append(&mut self.current_anon_lookups);
     }
 
-    /// returns the index of the appropriate lookup in anon_lookups
+    /// Find or create an anonymous lookup that meets a given condition
+    ///
+    /// This is a helper called by various methods that add anonymous lookup
+    /// rules.
+    ///
+    /// The `predicate` argument is a closure that should return `true` if a
+    /// given lookup can be used by the caller; if no suitable lookup is found,
+    /// then `make_lookup` will be called to create one.
     #[must_use]
     fn find_or_create_anon_lookup(
         &mut self,
-        check_fn: impl Fn(&T) -> bool,
-        new_fn: impl FnOnce(LookupFlag, Option<FilterSetId>) -> T,
+        predicate: impl Fn(&T) -> bool,
```

Comment:
Optional, I much prefer the naming style of `make_lookup`: it actually gives me some idea of what it is. `can_use_lookup` or `can_use_lookup_pred` would be more informative than just `predicate`.